### ISSUE TITLE
setFindTimeout() errors with "Invalid timeout type specified: ms"

### ISSUE THREAD

**Issue by mgroenhoff** [theintern/leadfoot#134]
If i override `session.capabilities.usesWebDriverTimeouts = true;` i do not get this error.
```
Creating session failed:  { [POST http://localhost:4444/wd/hub/session/c8ccd72b-a62f-488b-ac6d-c897c51861d1/timeouts / {"type":"implicit","ms":30000}] Invalid timeout type specified: ms
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:32.194Z'
System info: host: 'PC65', ip: '10.0.0.172', os.name: 'Windows 10', os.arch: 'x86', os.version: '10.0', java.version: '1.8.0_151'
Driver info: driver.version: unknown
    at Server.post (\node_modules\@theintern\src\Server.ts:367:15)
    at runRequest (\node_modules\@theintern\src\Session.ts:132:14)
    at Task_1.default.cancelled (\node_modules\@theintern\src\Session.ts:164:39)
    at new Task (\node_modules\@dojo\core\0.3.0\async\Task.ts:238:4)
    at Session._delegateToServer (\node_modules\@theintern\src\Session.ts:110:10)
    at Session.serverPost (\node_modules\@theintern\src\Session.ts:174:15)
    at Session.setTimeout (\node_modules\@theintern\src\Session.ts:225:24)
    at Session.setFindTimeout (\node_modules\@theintern\src\Session.ts:2080:15)
request: {
	url: 'http://localhost:4444/wd/hub/session/d8bddfa0-ad15-4c13-b7d6-a54c266545c0/timeouts',
	method: 'POST',
	data: { type: 'implicit', ms: 30000 }
}
response: {
	downloadBody: true,
	headers: { map: [Object] },
	status: 500,
	ok: false,
	statusText: 'Server Error'
}
```
Selenium standalone server: 3.8.1
IE (ia32) driver: 3.8.0
@theintern/leadfoot: 2.0.3

**Update:** Added stack trace, request and response

**Comment by jason0x43** [comment]
Do you have fixSessionCapabilities set to 'false' or 'no-detect'?

**Comment by mgroenhoff** [comment]
I had to look that up but no, we left it at it's default value and it is true

**Comment by jason0x43** [comment]
It's not weird that the driver version is unknown. Selenium doesn't reporter its version, and individual drivers may or may not report their versions. That's one of the reasons Leadfoot uses feature tests to determine whether or not WebDriver semantics are in use.

It looks like IEDriver is giving a Server Error where we're expecting something like a NoSuchCommand error. Assuming that's it, we can update the feature test to handle that case more intelligently.

**Comment by jason0x43** [comment]
This was fixed in 18c31b3c
